backwards-compat-tests/slt/nexmark-backwards-compat/validate_restart.slt

```diff
@@ -1,9 +1,17 @@
 include ../nexmark/test_mv_result.slt.part
 
 include ./delete.slt.part
+
+statement ok
+flush;
```

Comment:
If I don't add flush, q0,1,2 succeed, but q3 failed. Why? 🤡

Reply:
If it's ordering of `DELETE` and `INSERT`, I don't understand why  q0,1,2 succeed

Reply:
q0,1,2 seem only depend on `bid` while q3 also depends on `auction`

Reply:
So what does this imply? 🤔

Reply:
IDK, just seems to be explainable if `bid` table is OK but `auction` table is not ready

Reply:
I can reproduce this locally. It fails sometimes, and `person` is empty when it fails.

```
include ../e2e_test/nexmark/create_tables.slt.part

include ../e2e_test/nexmark/insert_person.slt.part
include ../e2e_test/nexmark/insert_auction.slt.part
include ../e2e_test/nexmark/insert_bid.slt.part

include ../e2e_test/streaming/nexmark/views/q3.slt.part
include ../e2e_test/streaming/nexmark/q3.slt.part

statement ok
DELETE FROM person;

statement ok
DELETE FROM auction;

statement ok
DELETE FROM bid;

include ../e2e_test/nexmark/insert_person.slt.part
include ../e2e_test/nexmark/insert_auction.slt.part
include ../e2e_test/nexmark/insert_bid.slt.part

statement ok
flush;

include ../e2e_test/streaming/nexmark/q3.slt.part

statement ok
drop materialized view nexmark_q3;

include ../e2e_test/nexmark/drop_tables.slt.part

```

Reply:
I had thought that when DML query returns, it will take effect, although not immediately visible. But it seems not correct? :thinking:

Reply:
After the DML returns, the data is only sent to some internal channels, and there are multiple channels so the order is not known.